extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -211,6 +205,28 @@ class AmpYoutube extends AMP.BaseElement {
     }
   }
 
+  /** @override */
+  mutatedAttributesCallback(mutations) {
+    if (mutations['data-videoid'] !== undefined) {
+      this.videoid_ = this.getVideoId_();
+      if (this.iframe_) { // `null` if element hasn't been laid out yet.
```

Comment:
I actually moved the `this.playerReadyPromise_.then` to within `sendCommand_` to reduce duplication. I assume that we never want to call `sendCommand_` before the player is ready as you mentioned.